packages/@lwc/shared/src/api-version.ts

```diff
@@ -12,19 +12,21 @@ export const enum APIVersion {
     V59_246_WINTER_24 = 59,
     V60_248_SPRING_24 = 60,
     V61_250_SUMMER_24 = 61,
+    V62_252_WINTER_25 = 62,
 }
 
 // These must be updated when the enum is updated.
 // It's a bit annoying to do have to do this manually, but this makes the file tree-shakeable,
 // passing the `verify-treeshakeable.js` test.
 
 export const LOWEST_API_VERSION = APIVersion.V58_244_SUMMER_23;
-export const HIGHEST_API_VERSION = APIVersion.V61_250_SUMMER_24;
+export const HIGHEST_API_VERSION = APIVersion.V62_252_WINTER_25;
```

Comment:
```ts
export const LOWEST_API_VERSION = allVersions.at(0)!;
export const HIGHEST_API_VERSION = allVersions.at(-1)!;
```
This still passes the tree shakeable test, but means we don't need to update every time (assuming that `allVersions` is always correctly ordered).

Reply:
Sadly we still support Safari v14, and [Array.prototype.at](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at#browser_compatibility) is not supported there.

We could probably do this with `[length - 1]` and maybe a `PURE` comment though.

Reply:
Done: 9b5c4ef18